frontend/src/views/Game.tsx

```diff
@@ -355,6 +376,20 @@ function GamePage() {
       });
   };
 
+  const nextProblem = () => {
+    setCurrentProblem((currentProblem.valueOf() + 1) % problems?.length);
+  };
+
+  const previousProblem = () => {
+    let temp = currentProblem.valueOf() - 1;
+
+    if (temp < 0) {
+      temp += problems?.length;
+    }
+
+    setCurrentProblem(temp);
+  };
```

Comment:
I know you didn't want me to comment on this, but I'll say one thing - I would recommend that you don't have it wrap around to problem 1 after clicking "Next" on problem 4 (out of 4 problems). This could make sense if you have the annotation 1/4, 2/4, etc., but if we don't then this functionality shouldn't exist.